src/Plant/struct/struct_tree_root.jl

```diff
@@ -1,4 +1,12 @@
-# struct for root layer
+"""
+    struct RootLayer{FT}
+
+# Arguments
+- `FT`    Floating type for the RootLayer
+
+# Description
+Calling RootLayer{FT}() will create one RootLayer struct for the root system.
+"""
```

Comment:
```suggestion
"""
    struct RootLayer{FT}

# Arguments
- `FT`    Floating type for the RootLayer

# Description
Calling RootLayer{FT}() will create one RootLayer struct for the root system.

# Fields
$(DocStringExtensions.FIELDS)
"""
```
Here, `FT` is a [type parameter](https://docs.julialang.org/en/v1/manual/types/#Parametric-Types-1). For structs, we should probably document the arguments, and this can be done nicely with `DocStringExtensions`, you can see an example [here](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Common/MoistThermodynamics/states.jl#L55-L80)

Reply:
You'll need to modify the doc strings in the structs, too